docs/deps_and_runner.md

```diff
@@ -1,19 +1,32 @@
+
 # Dependency manager
+### Requisitos
+- facil de usar
+- cuánto más se automatice, mejor
+- frecuente actividad de mantenimiento del proyecto
```

Comment:
Se te ha olvidado el principal: que entienda pyproject.tom. pip no lo hace.
Por otro lado, no sé quién ha pedido que elijáis un gestor de dependencias. Lo que hay que elegir es un task runner.